inst/shiny/app.R

```diff
@@ -0,0 +1,28 @@
+library(shiny)
+library(tidyverse)
+library(newbullwhipgame)
+
+ui <- shinyUI(
+  navbarPage(
+    "newbullwhipgame",
+    id = "mainNavbarPage",
+    theme = "flatky.css",
+    uiModDesc("Description"),
+    uiModPlay("Play"),
+    uiModGlossary("Glossary"),
+    uiModAbout("About")
+  ))
+
+server <- shinyServer(function(input, output, session) {
+  ns <- session$ns
+
+  # reactive values
+  inputs_rv <- reactiveVal()

```

Comment:
What is the role of `inputs_rv <- reactiveVal()` here and in srvModPlay.R file ?